packages/smooth_app/lib/pages/product/edit_ingredients_page.dart

```diff
@@ -178,7 +178,26 @@ class _EditOcrPageState extends State<EditOcrPage> {
           ),
         );
       } else {
-        children.add(Container(color: Colors.white));
+        children.add(
+          Container(
+            alignment: Alignment.center,
+            margin: EdgeInsets.only(bottom: size.height * 0.25),
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              crossAxisAlignment: CrossAxisAlignment.center,
+              children: <Widget>[
+                const Icon(
+                  Icons.image_not_supported,
+                  size: 200,
+                ),
+                Text(
+                  'Upload an image to automatically extract the provided information',
+                  style: Theme.of(context).textTheme.bodySmall,
```

Comment:
We're not very consistent in the app regarding text size, that's a fact.
This line won't help, as we _never_ used `bodySmall` before. If you could find another place in the app where we use "small texts" and use the same size parameter, that would make sense.